packages/website/versioned_docs/version-legacy/inside-the-engine.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Inside the engine
+---
+
+This page explains in more detail how the crawler extracts content from your page, and how it ranks the results.
+
+## Crawling
+
+Each crawl begins its journey at `start_urls` value specified in your config. It will read those pages, recursively extract and follow every link in those pages until it has browsed every compliant page.
+
+If you have explicitly defined a `sitemap.xml`, our crawler will scrape every provided and compliant page. We do recommend using [a sitemap][1] since it explicitly exposes URLs to crawl and avoid missing pages that aren't linked from another one.
```

Comment:
```suggestion
If you have explicitly defined a `sitemap.xml`, our crawler will scrape every provided and compliant page. We recommend using [a sitemap][1] since it explicitly exposes URLs to crawl and avoid missing pages that aren't linked from another page.
```